### PR TITLE
Telegram Bot Action

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by m-dad — APPROVED** [review]

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -20,3 +20,29 @@ jobs:
...
+
+            📌 <b>Release Tag</b>: ${{ github.event.release.tag_name }}
+            📆 <b>Release Date</b>: ${{ steps.current-date.outputs.now }}
+            🧔 Release Creator: <a href="${{ github.event.release.author.html_url }}">${{ github.actor }}</a>
+
+            🔴 <b>Release Title</b>: ${{ github.event.release.name }}
+
+            📎 Visit for more details: ${{ github.event.release.html_url }}
+
+            📍 Also Visit <a href="${{ github.event.repository.html_url }}">The Repository</a>!
```
Comment (m-davoodi) [inline .github/workflows/publish.yml]: add a new line at the end of the file

**Review by m-davoodi — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (ghost) [reply .github/workflows/publish.yml]: Done

**Review by m-davoodi — APPROVED** [review]

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -20,3 +20,29 @@ jobs:
       - run: yarn changeset publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  send-telegram-message:
```
Comment (mh-alahdadian) [inline .github/workflows/publish.yml:23]: no separate job , add its steps to publish job

**Review by mh-alahdadian — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 2 above)
Comment (ghost) [reply .github/workflows/publish.yml:23]: it is more clean!

**Review by mh-alahdadian — APPROVED** [review]

**Pull request merged**